src/OmniSharp/Models/ProjectStatusMessage.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace OmniSharp.Models
+{
+    public class ProjectStatusMessage
+    {
+        public string FileName { get; set; }
+        public string LogLevel { get; set; }
+        public string Text { get; set; }
+        public string ExceptionMessage { get; set; }
+        public IEnumerable<DiagnosticLocation> Diagnostics { get; set; }
```

Comment:
It's the type we send out for diagnostics: https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp/Api/Diagnostics/OmnisharpController.Diagnostics.cs#L46